lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
Ah, true, `publish` does implicitly attach.

See https://github.com/ably/ably-ruby/commit/c5292f8738d166b96e911e2eced794b1dbda7270 for a fix to the docs.